app/src/main/java/com/dessalines/thumbkey/db/AppDb.kt

```diff
@@ -48,7 +48,7 @@ const val DEFAULT_HIDE_SYMBOLS = 0
 const val DEFAULT_KEY_BORDERS = 1
 const val DEFAULT_SPACEBAR_MULTITAPS = 1
 const val DEFAULT_SLIDE_SENSITIVITY = 9
-const val DEFAULT_SLIDE_ENABLED = 0
+const val DEFAULT_SLIDE_ENABLED = 1
```

Comment:
Unfortunately android sqlite does not let you change defaults on columns, even with a migration. You have to use a temp table, then regenerate the entire table in a migration. 

But also, I'm not sure that I'd want slide swipe to be enabled by default.

Reply:
Oh I see, I didn't know this. Shall I open another PR to keep the commit history clean, or just add a commit to this that keeps slide as disabled? I'm happy to keep it as disabled if that's your preference :).

Reply:
The slide is currently disabled already by default... but I'm open to making it enabled by default, given the really cool features that @storvik has added. 

You'll need to add a migration to : 

- Create a temp table
- Insert the data into the temp table with the new default
- Delete the old table.
- Rename the temp table to the old name.

Reply:
Well that sums up programming pretty well.
Me: "Hey Dess, I wanna change this variable from 0 to 1"
Dessalines: "Okay mate, just migrate the database, create a temporary table, insert data, replace the old one." :laughing: 

Once we had swiping to select and delete text, that made me think the feature should be enabled by default as it's such a useful feature.

I'll work on this in a bit :smile: .

Reply:
Haha no probs. Its an annoying limitation of sqlite that you can't change default values on columns.